### PR TITLE
[Refactor] Rename namespaces (#71)

### PR DESCRIPTION
**Pull request by chrisw000**
Rename namespace from GDAXClient GDAXSharp
also renames the project folders etc. so projects will need to be referenced

This PR replaces #76

### CODE REVIEW

**Comment by chrisw000** [comment]
I closed #76 as the original branch I had got screwed up merge wise....

I've just noticed #79 has also been opened since, for same thing.... there is a subtle difference in file count / change count.

Take your pick!

**Comment by confessore** [comment]
what are the extra deletions? it should have been a 1:1 additions:deletions

also, I forgot to update the readme

**Comment by chrisw000** [comment]
the are 2 extra deletions from GDAXSharp.Services.Withdrawals.Models.Coinbase.cs

an unneeded using System; and blank line have gone....

File: GDAXSharp.Specs/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -4,11 +4,11 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("GDAXClient.Specs")]
+[assembly: AssemblyTitle("GDAXSharp.Specs")]
```
Comment (dougdellolio) [inline GDAXSharp.Specs/Properties/AssemblyInfo.cs]: 👍

**Pull request merged**